docs/source/recipes.rst

```diff
@@ -0,0 +1,399 @@
+
+      >>> f3 = cf.read('monthly_precipitation.nc')
+      >>> print(f3)
+      [<CF Field: long_name=precipitation(long_name=time(1452), latitude(144), longitude(192)) mm/month>]
+
+3. Select near surface temperature by index and look at its contents:
+
+   .. code-block:: python
+
+      >>> v = f1[0]
+      >>> print(v)
+      Field: northward_wind (ncvar%vas)
+      ---------------------------------
+      Data            : northward_wind(time(1980), latitude(144), longitude(192)) m s-1
+      Cell methods    : area: time(1980): mean
+      Dimension coords: time(1980) = [1850-01-16 00:00:00, ..., 2014-12-16 00:00:00] 360_day
+                      : latitude(144) = [-89.375, ..., 89.375] degrees_north
+                      : longitude(192) = [0.0, ..., 358.125] degrees_east
+                      : height(1) = [10.0] m
+
+      >>> u = f2[0]
+      >>> print(u)
+      Field: eastward_wind (ncvar%uas)
+      --------------------------------
+      Data            : eastward_wind(time(1980), latitude(144), longitude(192)) m s-1
+      Cell methods    : area: time(1980): mean
+      Dimension coords: time(1980) = [1850-01-16 00:00:00, ..., 2014-12-16 00:00:00] 360_day
+                      : latitude(144) = [-89.375, ..., 89.375] degrees_north
+                      : longitude(192) = [0.0, ..., 358.125] degrees_east
+                      : height(1) = [10.0] m
+
+      >>> pre = f3[0]
+      >>> print(pre)
+      Field: long_name=precipitation (ncvar%pre)
+      ------------------------------------------
+      Data            : long_name=precipitation(long_name=time(1452), latitude(144), longitude(192)) mm/month
+      Dimension coords: long_name=time(1452) = [1901-01-16 00:00:00, ..., 2021-12-16 00:00:00] gregorian
+                      : latitude(144) = [-89.375, ..., 89.375] degrees_north
+                      : longitude(192) = [0.0, ..., 358.125] degrees_east
+
+4. Plot the northward and eastward wind components over precipitation data for June 1995 by creating a subspace (`~cf.Field.subspace`) over a date-time object (`cf.dt`) using `cfplot.con <http://ajheaps.github.io/cf-plot/con.html>`_. Here `cfplot.gopen <http://ajheaps.github.io/cf-plot/gopen.html>`_ is used to define the parts of the plot area, which is closed by `cfplot.gclose <http://ajheaps.github.io/cf-plot/gclose.html>`_, `cfplot.cscale <http://ajheaps.github.io/cf-plot/cscale.html>`_ is used to choose one of the colour maps amongst many available,`cfplot.levs <http://ajheaps.github.io/cf-plot/levs.html>`_ is used to set the contour levels for precipitation data and `cfplot.vect <http://ajheaps.github.io/cf-plot/vect.html>`_ is used to plot the wind vectors for June 1995:
+
+   .. code-block:: python
+
+      >>> cfp.gopen()
+      >>> cfp.cscale('precip4_11lev')
+      >>> cfp.levs(step=100)
+      >>> cfp.con(pre.subspace(T=cf.dt('1995-06-16')), lines=False, title = 'June 1995 monthly global precipitation')
```

Comment:
Could we prevent the long-line scrolling on this and the next line?

Reply:
could to `>>> june_95 = cf.dt('1995-06-16'))` and then `subspace(T=june_95)` three times?

Or per (better?) `>>> june_95 = cf.year(1995) & cf.month(6)`. Could be better because  it doesn't rely on knowing where the representative day of the months is (it's 16 in this case, but presumably could be 15 for Feb 95)

Reply:
When a subspace is created after setting june_95 using ` >>> june_95 = cf.year(1995) & cf.month(6)`, it raises a ValueError `”Can’t create date-time object with unknown calendar {calendar!r}”` and a TypeError: `an integer is required`. I also tried adding a `calendar='360_day'` to `cf.dt` but still the samre result.